fossology/fossology
# fix(delagent): Prevent unauthorized delete from CLI

**How to test**
1.  Upload a package with 'user1'.
2.  Get the upload_pk of the new upload.
3.  Create a user 'user2' with read-only access.
4.  From CLI delete the upload as user2 `./delagent --user user2 --password 123 -U <upload_pk>`. Should see an error.
5.  From CLI delete the upload as user1 `./delagent --user user1 --password 123 -U <upload_pk>`. Should work fine.

Closes #186.
